frontend/public/components/machine-set.tsx

```diff
@@ -304,12 +308,95 @@ export const MachineSetList: React.SFC = (props) => {
         props: { className: tableColumnClasses[2] },
       },
       {
-        title: '',
+        title: t('public~Instance type'),
         props: { className: tableColumnClasses[3] },
       },
+      {
+        title: t('public~CPU cores'),
+        props: { className: tableColumnClasses[4] },
+      },
+      {
+        title: t('public~Memory'),
+        props: { className: tableColumnClasses[5] },
+      },
+      {
+        title: '',
+        props: { className: tableColumnClasses[6] },
+      },
     ];
   };
 
+  const MachineSetTableRow: React.FC<RowFunctionArgs<MachineSetKind>> = ({ obj }) => {
+    const relatedMachines =
+      machinesLoaded &&
+      machines?.filter(
+        (machine) =>
+          machine.metadata.labels?.['machine.openshift.io/cluster-api-machineset'] ===
+          obj.metadata.name,
+      );
+    const relatedNodes =
+      nodesLoaded && relatedMachines.length > 0
+        ? nodes.filter((node) =>
+            relatedMachines.some((machine) => node.metadata.uid === machine.status?.nodeRef?.uid),
+          )
+        : [];
+
+    return (
+      <>
+        <TableData className={tableColumnClasses[0]}>
+          <ResourceLink
+            kind={machineSetReference}
+            name={obj.metadata.name}
+            namespace={obj.metadata.namespace}
+          />
+        </TableData>
+        <TableData
+          className={classNames(tableColumnClasses[1], 'co-break-word')}
+          columnID="namespace"
+        >
+          <ResourceLink kind="Namespace" name={obj.metadata.namespace} />
+        </TableData>
+        <TableData className={tableColumnClasses[2]}>
+          <Link
+            to={`${resourcePath(
+              machineSetReference,
+              obj.metadata.name,
+              obj.metadata.namespace,
+            )}/machines`}
+          >
+            {t('public~{numReadReplicas} of {numDesiredReplicas} machine', {
+              numReadyReplicas: getReadyReplicas(obj),
+              numDesiredReplicas: getDesiredReplicas(obj),
+              count: getDesiredReplicas(obj),
+            })}
+          </Link>
+        </TableData>
+        <TableData className={tableColumnClasses[3]}>
+          {getMachineSetInstanceType(obj) || '-'}
+        </TableData>
+        <TableData className={tableColumnClasses[4]}>
+          {relatedNodes.reduce(
+            (cpuSize, node) => cpuSize + _.toInteger(convertToBaseValue(node.status?.capacity.cpu)),
```

Comment:
I think this reads better. `_.toIntegar` makes me think we need to convert from a string (which isn't the case) and seems unnecessary just to check for undefined/null.

```suggestion
            (cpuSize, node) => cpuSize + (convertToBaseValue(nnode.status?.capacity.cpu) ?? 0),
```